TYPO3.Neos/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -104,6 +104,18 @@ additionalProperties:
 
                   'editorOptions': { type: dictionary, description: 'options for the given editor' }
 
+                  'editorListeners':
+                    type: dictionary
+                    additionalProperties:
+                      type: dictionary
+                      properties:
+                        'property': { type: ['string'], description: 'The name of an observed property in the same NodeType.' }
+                        'handler': { type: ['string'], description: 'The path to a handler JavaScript object, similar to custom editors and validators.' }
+                        'handlerOptions':
+                          type: dictionary
+                          additionalProperties:
+                            type: ['string', 'null', 'dictionary', 'boolean', 'array']
```

Comment:
not sure if these lines are needed? they aren't present for editorOptions in line 105 e.g.